### PR TITLE
SAC Documentation - Benchmarks - Minor code tweaks

### PR DESCRIPTION
**Pull request by dosssman**
## Description

- SAC documentation prototype
- SAC `qf_loss` computation: removed the `/2` gradient scaling so that .backward() is more aligned with the theory. Instead, log `qf_loss` = (qf1_loss + qf2_loss) / 2.` for meaningful comparison with mono Q-value network algorithms.
Same is done in [OpenAI SpinningUP](https://github.com/openai/spinningup/blob/038665d62d569055401d91856abb287263096178/spinup/algos/pytorch/sac/sac.py#L199) for example.
- Added benchmark instructions to run SAC on Mujoco and pybullet environments
- Added SAC runs for 6 continuous control envs (3 Mujoco, 3 PyBullet) to cleanrl and corresponding openrlbenchmark reports.

## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] New algorithm
- [x] Documentation
- [x] Benchmarks

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/vwxyzjn/cleanrl/blob/master/CONTRIBUTING.md) guide (**required**).
- [x] I have ensured `pre-commit run --all-files` passes (**required**).
- [x] I have updated the documentation and previewed the changes via `mkdocs serve`.
- [ ] ~I have updated the tests accordingly (if applicable).~

If you are adding new algorithms or your change could result in performance difference, you may need to (re-)run tracked experiments. See https://github.com/vwxyzjn/cleanrl/pull/137 as an example PR. 
- [x] I have contacted @vwxyzjn to obtain access to the [openrlbenchmark W&B team](https://wandb.ai/openrlbenchmark) (**required**).
- [x] I have tracked applicable experiments in [openrlbenchmark/cleanrl](https://wandb.ai/openrlbenchmark/cleanrl) with `--capture-video` flag toggled on (**required**).
- [x] I have added additional documentation and previewed the changes via `mkdocs serve`.
    - [x] I have explained note-worthy implementation details.
    - [x] I have explained the logged metrics.
    - [x] I have added links to the original paper and related papers (if applicable).
    - [x] I have added links to the PR related to the algorithm.
    - [x] I have created a table comparing my results against those from reputable sources (i.e., the original paper or other reference implementation).
    - [x] I have added the learning curves (in PNG format with `width=500` and `height=300`).
    - [x] I have added links to the tracked experiments.
- [ ] ~I have updated the tests accordingly (if applicable).~

### CODE REVIEW

**Comment by vwxyzjn** [comment]
Thank you @dosssman. This is a really high-quality benchmark. I have asked @ikostrikov, who maintains https://github.com/ikostrikov/jaxrl, to help review this PR. Thanks @ikostrikov!

File: docs/rl-algorithms/sac.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
+# Soft Actor-Critic (SAC)
+
+## Overview
+
+The Soft Actor-Critic (SAC) algorithm extends the DDPG algorithms by 1) using a stochastic policy, which in theory can express multi-modal optimal policies.
```
Comment (vwxyzjn) [inline docs/rl-algorithms/sac.md:5]: DDPG algorithms > DDPG algorithm

File: docs/rl-algorithms/sac.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            target_entropy = -torch.prod(torch.Tensor(envs.single_action_space.shape).to(device)).item()
+            log_alpha = torch.zeros(1, requires_grad=True, device=device)
+            alpha = log_alpha.exp().item()
+            a_optimizer = optim.Adam([log_alpha], lr=args.q_lr)
+        else:
+            alpha = args.alpha
+    ```
+
+3. [`sac_continuous_action.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/sac_continuous_action.py) uses `--batch-size=256` while :material-github: [openai/spinningup](https://github.com/openai/spinningup/blob/038665d62d569055401d91856abb287263096178/spinup/algos/tf1/sac/sac.py#L44)'s uses `--batch-size=100` by default.
+
```
Comment (vwxyzjn) [inline docs/rl-algorithms/sac.md:185]: I'd also mention that we have used a global gradient clipping which is an optimization detail.

**Review by vwxyzjn — CHANGES_REQUESTED** [review]
One other thing. Would you mind customizing the chart a bit like the other charts in the benchmark? Use `CleanRL's sac_continuous_action.py` instead of `exp_name: sac_continuous_action` for the legend. I'd also change the line color to red for consistency.

Everything else looks good :) Feel free to merge once you have a chance to address these .

**Comment by dosssman** [comment]
- [x] Change the legend of the SAC experiments
- [x] Change the color of the SAC experiments in the standalone Mujoco and PyBullet reports
- [x] Re-export the SAC plots with the red color
- [x] Fixed the "DDPG algorihtms" typo
- [x] Mention global gradient clipping

File: docs/rl-algorithms/sac.md (hunk 1 above)
Comment (dosssman) [reply docs/rl-algorithms/sac.md]: Thanks. Fixed.

File: docs/rl-algorithms/sac.md (hunk 2 above)
Comment (dosssman) [reply docs/rl-algorithms/sac.md:185]: Done, but maybe too extensively ?

**Comment by dosssman** [comment]
Oh man, I finally found the last reference for SAC I was looking for but could not remember: [pranz24/pytorch-soft-actor-critic](https://github.com/pranz24/pytorch-soft-actor-critic) .

**Review by vwxyzjn — APPROVED** [review]
It looks really good now. Thank you @dosssman and feel free to merge.

**Comment by dosssman** [comment]
Thanks for the review. Merging then.
